### DISCUSSION TITLE
Element API

### DISCUSSION THREAD

**Discussion by draber** [mdn/mdn-community#526] [Content]
I have been looking for an element API but it seems there is none. I wrote [a bit of code](https://github.com/draber/mdn-tag-collector) that parses this repo and solves my particular problem. If you believe you could benefit from it in one form or another, please help yourselves. 

Mind you that what I wrote is pretty rough, as far as I'm concerned this was just a quick one-off.

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
